lib/logger-wrapper.ts

```diff
@@ -0,0 +1,38 @@
+import { Logger } from "tslog";
+
+export class LoggerWrapper {
+  private logger: Logger;
+  private on: boolean = true;
+  constructor(logger: Logger) {
+    this.logger = logger;
+  }
+  public logOn(): LoggerWrapper {
+    this.on = true;
+    return this;
+  }
+  public logOff(): LoggerWrapper {
+    this.on = false;
+    return this;
+  }
+  public silly(...args: unknown[]): void {
+    this.on ? this.logger.silly(...args) : null;
+  }
+  public trace(...args: unknown[]): void {
+    this.on ? this.logger.trace(...args) : null;
+  }
+  public debug(...args: unknown[]): void {
+    this.on ? this.logger.debug(...args) : null;
+  }
+  public info(...args: unknown[]): void {
+    this.on ? this.logger.info(...args) : null;
+  }
+  public warn(...args: unknown[]): void {
+    this.on ? this.logger.warn(...args) : null;
+  }
+  public error(...args: unknown[]): void {
+    this.on ? this.logger.error(...args) : null;
+  }
+  public fatal(...args: unknown[]): void {
+    this.on ? this.logger.fatal(...args) : null;
+  }
+}
```

Comment:
please read https://github.com/line/line-blockchain-developers-sdk-js/pull/86#discussion_r1060395829

Reply:
comment is reflected on : https://github.com/line/line-blockchain-developers-sdk-js/pull/86/commits/b824677bc2b8115d523a9bbd1d97f95fec80250c